data/Kocacinar_2003/metadata.yml

```diff
@@ -0,0 +1,248 @@
+    notes: no contact
+  austraits_curators: Elizabeth Wenk
+dataset:
+  data_is_long_format: no
+  custom_R_code:         '
+    data %>% mutate(location = "University of Toronto, Ontario")
+  '
+  collection_date: unknown/2003
+  taxon_name: taxon_name
+  location_name: location
+  description: A comparison of stem hydraulic conductivity and vascular anatomy between
+    eight C3 and eight C4 herbaceous species.
+  basis_of_record: experiment
+  life_stage: saplings
+  sampling_strategy: All plants were grown from seeds for three months in 6 L pots
+    on an outdoor roof at the University of Toronto, Ontario. The potting medium was
+    a mix of 25% Promix (Sun Gro Horticulture Canada Ltd, Seba Beach, AB, Canada),
+    25% sand and 50% topsoil. The plants were watered regularly to avoid drought and
+    were fertilized once a week with a full-strength Hoagland's solution. Plants were
+    measured approximately 3 months after germination.
+  original_file: none, transcribed from paper
+  notes: Only the species with Australian seed sources are included.
+locations:
+  University of Toronto, Ontario:
+    latitude (deg): 43.67
+    longitude (deg): -79.42
+contexts:
+- context_property: seed provenance
+  category: treatment
+  var_in: locality
+  values:
+  - value: Australian National University
+    description: Seeds sourced from plants grown at ANU.
+  - value: Darwin, Australia (12°15' S 131°10' E)
+    description: Seeds sourced from an area about 80 km north of Darwin, Australia
+      (12°15' S 131°10' E)
+traits:
+- var_in: stem_vessel_diameter
+  unit_in: um
+  trait_name: stem_vessel_diameter
+  entity_type: population
+  value_type: mean
+  basis_of_value: measurement
+  replicates: '6'
+  methods: Xylem area and conduit diameters were measured on stem cross-sections obtained
+    from the same samples as used for hydraulic conductivity measurements. For each  pecies,
+    six of the stem segments that had previously been used for Kh determination were
+    fixed with formalin, acetic acid and ethyl alchohol in water (FAA), paraffin-embedded
+    and sectioned using a sliding microtome. Sections were examined under a light
+    microscope and images analysed with imaging software (Image Pro Plus; Carsen Group
+    Inc., Markham, Ontario, Canada). The lengths of major and minor axes across a
+    conduit were averaged to obtain vessel diameters. Maximum vessel diameter (MVD)
+    was measured from each cross-section on the largest vessel. The vessels mean diameter
+    (VMD), hydraulically weighted mean diameter (HMD), mean diameter of conduits that
+    facilitate 95% of the flow (D95) and vessel frequency (VF, the number of vessels
+    per xylem area) were determined for each segment by measuring all xylem conduits
+    in three randomly chosen sectors from the cambium to pith that together represented
+    at least 20% of the stem xylem area. The hydraulically weighted mean diameter
```

Comment:
Is it worth deleting the sentences from here onwards to condense the methods a bit? (And same for the rest of the traits?)

Reply:
done